awx/ui/client/features/output/details.component.js

```diff
@@ -113,6 +113,18 @@ function getVerbosityDetails () {
     return { label, value };
 }
 
+function getEnvironmentDetails () {
```

Comment:
since this gets called in our `subscribe` function down below, we'd want to add an arg here for the virtualenv obj. If a value is passed to the function it should override the one retrieved from `resource.model` since it is newer.